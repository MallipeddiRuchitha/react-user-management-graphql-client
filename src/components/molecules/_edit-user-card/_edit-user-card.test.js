import React from 'react';
import { create } from "react-test-renderer";
import { render, fireEvent, getByRole } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import EditUserCard from './_edit-user-card';




describe("EditUserCard component",() =>{
    
     
    test("Matches the snapshot",() => {
        const editUserCard = create(<EditUserCard />);
        expect(editUserCard.toJSON()).toMatchSnapshot();

    })
    test("EditUserCard element is rendered",() => {
        const { getByTestId, getByText } = render(<EditUserCard />);
        expect(getByTestId("editUserCard")).toBeInTheDocument();
        
    });
  
   
   
    
});