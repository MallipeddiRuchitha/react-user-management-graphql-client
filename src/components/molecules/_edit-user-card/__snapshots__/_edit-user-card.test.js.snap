// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditUserCard component Matches the snapshot 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-styleCard-3 MuiPaper-elevation1 MuiPaper-rounded"
  data-testid="editUserCard"
>
  <div
    data-testid="input"
  >
    <label
      className="makeStyles-styleLabel-4"
    />
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline"
      onClick={[Function]}
    >
      <input
        className="MuiInputBase-input MuiInput-input"
        data-testid="value"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="text"
      />
    </div>
  </div>
  <div
    data-testid="input"
  >
    <label
      className="makeStyles-styleLabel-4"
    />
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline"
      onClick={[Function]}
    >
      <input
        className="MuiInputBase-input MuiInput-input"
        data-testid="value"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="text"
      />
    </div>
  </div>
  <div
    data-testid="button"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      />
    </button>
  </div>
</div>
`;
